### DISCUSSION TITLE
Electrical component for output load and energy in sub

### DISCUSSION THREAD

**Discussion by Gast9ra** [FakeFishGames/Barotrauma#6794] [Ideas and suggestions]
Electrical component for outputting load and energy. Or add load and energy output to the Junction Box. To use these values in other components.

**Comment by Ek-Videogames** [comment]
A weird workaround (but sub-editor only) is to put a invincible-uninteractable-hiddeningame nuclear reactor on the circuit you want to monitor - but yeah +1 we really could use this.
